doc/api/esm.md

```diff
@@ -808,6 +808,31 @@ const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 ```
 
+### No `require.resolve`
+
+Former use cases relying on `require.resolve` to determine the resolved path
+of a module can be supported via `import.meta.resolve`:
+
+```js
+(async () => {
+  const dependencyAsset = await import.meta.resolve('component-lib/asset.css');
+})();
+```
+
+`import.meta.resolve` also accepts a second argument which is the parent module
```

Comment:
require.resolve doesn’t do this; perhaps we could make it do this, so the two APIs are parallel?

Reply:
It’s been supported since <https://github.com/nodejs/node/pull/16397> as:
```js
require.resolve('./dep', { paths: [ __filename ] });
```